### PR TITLE
start to the active directory / can't move a file to the folder where…

### PR DESCRIPTION
**Pull request by pvilalta**
Hello all!

I made same modifications into the moveTo window 
* The starting directory is now the active one, where the user is
* It's impossible to move a file into the folder where it is currently (see screenshot below)

Paul 

Screenshot 2021-11-15 at 15 34 52
Screenshot 2021-11-15 at 15 34 23

### CODE REVIEW

File: src/components/specific/files/folder-selector/FolderSelector.vue [hunk 1]
```diff
@@ -118,33 +121,50 @@ export default {
...
+            child.type === FILE_TYPES.FOLDER &&
+            !props.files.some(f => child.id === f.id) &&
+            (props.project.isAdmin ||
+              child.userPermission === FILE_PERMISSIONS.READ_WRITE)
+        )
+    );
+
+    const isAllowedToMoveFile = computed(() =>
+      props.files.some(f => {
+        if (selectedFolder.value) {
```
Comment (Amoki) [inline src/components/specific/files/folder-selector/FolderSelector.vue]: For branlette intellectuelle only, how could you replace this `if` with a fully functional method?

Comment (pvilalta) [reply src/components/specific/files/folder-selector/FolderSelector.vue]: here is my proposition 

```js
  const isAllowedToMoveFile = computed(() =>
    props.files.some(f =>
      selectedFolder.value
        ? selectedFolder.value.id === f.parentId
        : currentFolder.value.id === f.parentId
    )
  );
 ```
this code has been tested

Comment (Amoki) [reply src/components/specific/files/folder-selector/FolderSelector.vue]: This is still a condition. I meant without using any condition (no  `if` nor ternary)

Comment (Amoki) [reply src/components/specific/files/folder-selector/FolderSelector.vue]: Forget my previous comments.

The `if` is computed for each file and there is no way the `selectedFolder` value can change during the loop. You should set a variable before the loop.

**Review by Amoki — CHANGES_REQUESTED** [review]

**Review by Amoki — APPROVED** [review]

**Pull request merged**